src/eval_forecast_submissions.R

```diff
@@ -330,16 +383,16 @@ parser <-
     "to the COVID-19 Forecast Hub"
   )) |>
   argparser::add_argument(
-    "--scores-as-of",
+    "--scores_as_of",
```

Comment:
```suggestion
    "--scores_as_of",
```
In general, convention is underscores for positional args and hyphens for flag args.

So 

```
myscript.R positional_arg --flag-arg <value>
```

Reply:
argparser will then translate the hyphens to underscores